traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -1018,7 +1018,7 @@ func updateV31(w http.ResponseWriter, r *http.Request, inf *api.APIInfo, dsV31 *
 	}
 
 	if newDSType.HasSSLKeys() && (oldHostName != newHostName || oldCDNName != newCDNName || oldRoutingName != *ds.RoutingName) {
-		return nil, http.StatusForbidden, nil, errors.New("delivery service has ssl keys that cannot be automatically changed")
+		return nil, http.StatusBadRequest, errors.New("delivery service CDN is immutable"), errors.New("delivery service has ssl keys that cannot be automatically changed")
```

Comment:
is this supposed to return 2 errors? because i'm only seeing one.

Reply:
maybe the error message needs to be something like - "delivery service has ssl keys that cannot be automatically changed therefore, cdn and routing name are immutable"

Reply:
The error I added is user error. The system error in my opinion is not needed but I maybe wrong.

Reply:
Done.